postgresql/manifests/energy-platform/postgresql/upstream/postgres.yaml

```diff
@@ -0,0 +1,365 @@
+---
+# Source: postgresql/templates/secrets.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgresql
+  namespace: "energy-platform"
+  labels:
+    app.kubernetes.io/instance: postgresql
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: postgresql
+    app.kubernetes.io/version: 16.1.0
+    helm.sh/chart: postgresql-13.2.24
+  annotations:
+    app.uw.systems/description: postgresql for energy platform
+    app.uw.systems/repos: https://github.com/utilitywarehouse/shared-kustomize-bases/tree/main/postgresql
+    app.uw.systems/tier: tier_1
+type: Opaque
+data:
+  postgres-password: "T0hTMW5wMEFSSQ=="
```

Comment:
This is public repository :slightly_smiling_face: 
Should this secret rather be encrypted in kubernetes-manifests?  This can be done with parameter `auth.existingSecret`.